projects/fsastorefrontlib/src/cms-components/my-account/policy/policies-chart/policies-chart.component.ts

```diff
@@ -29,27 +29,36 @@ export class PoliciesChartComponent implements OnInit, OnDestroy {
     protected languageService: LanguageService,
     protected translation: TranslationService,
     protected chartConfig: ChartConfig
-  ) {}
+  ) {
+    this.subscription.add(
+      this.languageService
+        .getActive()
+        .pipe(
+          tap(lang => {
+            if (this.language && this.language !== lang) {
+              this.policyService.loadPolicies();
```

Comment:
Every time when language is changed `loadPolicies` will trigger `getPolicies` from `ngOnInit` and supply translated policies.